### PR TITLE
Fixes grammatical errors of mysiam.txt

### PR DESCRIPTION
**Pull request by rakshitkumarcse**

### CODE REVIEW

**Comment by rakshitkumarcse** [comment]
@an3l @iangilfillan
Kindly see #913 
Please merge.

File: Docs/myisam.txt [hunk 1]
```diff
@@ -833,9 +833,9 @@ Dynamic table characteristics
 * Each record is preceded by a bitmap indicating which columns are empty (`''') for string columns, or zero for numeric columns (this isn't the same as columns containing `NULL' values).  If a string column has a length of zero after removal of trailing spaces, or a numeric column has a value of zero, it is marked in the bit map and not saved to disk.  Non-empty strings are saved as a length byte plus the string contents. 
 * Usually takes much less disk space than fixed-length tables. 
 * Each record uses only as much space as is required. If a record becomes larger, it is split into as many pieces as required.  This results in record fragmentation. 
-* If you update a row with information that extends the row length, the row will be fragmented.  In this case, you may have to run `myisamchk -r' from time to time to get better performance.  Use `myisamchk -ei tbl_name' for some statistics. 
+* If you update a row with information that extends the row length, the row will be fragmented.  In this case, you may have to run `myisamchk -r' from time to time to get a better performance.  Use `myisamchk -ei tbl_name' for some statistics.
```
Comment (iangilfillan) [inline Docs/myisam.txt]: "get better performance" was correct, not "get a better performance"

File: Docs/myisam.txt [hunk 2]
```diff
@@ -899,3 +899,4 @@ int _mi_read_dynamic_record(MI_INFO *info, my_off_t filepos, byte *buf)
 Printed on 17/03/00
 
 C-7
+
```
Comment (iangilfillan) [inline Docs/myisam.txt]: No need for an extra line here

File: Docs/myisam.txt [hunk 3]
```diff
@@ -165,31 +165,31 @@ int mi_extra(MI_INFO *info, enum ha_extra_function function)
...
-HA_EXTRA_QUICK=1			Optimize for speed 
-HA_EXTRA_RESET=2			Reset database to after open 
-HA_EXTRA_CACHE=3			Cash record in HA_rrnd() 
-HA_EXTRA_NO_CACHE=4			End cacheing of records (def) 
-HA_EXTRA_NO_READCHECK=5		No readcheck on update 
-HA_EXTRA_READCHECK=6			Use readcheck (def) 
-HA_EXTRA_KEYREAD=7			Read only key to database 
-HA_EXTRA_NO_KEYREAD=8		Normal read of records (def) 
-HA_EXTRA_NO_USER_CHANGE=9	No user is allowed to write 
+    HA_EXTRA_NORMAL=0            Optimize for space (def)
```
Comment (iangilfillan) [inline Docs/myisam.txt]: No need for extra spacing at the start of the line

File: Docs/myisam.txt [hunk 4]
```diff
@@ -647,7 +647,7 @@ Zero if successful. Non-zero if an error occurred.
 EACCES
 The file was opened for read-only access.
 HA_ERR_RECORD_CHANGED
-	When mi_update() read the current record contents before updating, it differed from oldbuf.
+    When mi_update() read the current record contents before updating, it differed from oldbuf.
```
Comment (iangilfillan) [inline Docs/myisam.txt]: No need for extra spacing at the start of the line

File: Docs/myisam.txt [hunk 5]
```diff
@@ -60,9 +60,9 @@ See MySQL Appendix B "Choosing a table type".
 MyISAM allows about 32 indexes. However the official MySQL limit is 16 until MySQL 4.0.
```
Comment (iangilfillan) [inline Docs/myisam.txt]: The content of this document is extremely dated, referring to MySQL 3.23.5 as the current release etc. so perhaps @vuvova can comment on whether this document is still needed, or should, separately to this PR, be updated at some point.

**Comment by svoj** [comment]
Hi @rakshitkumarcse ,

Unfortunately we're unable to take care of your contribution without your action upon requested changes. Feel free to reopen and act accordingly.
 
Thanks,
Sergey

**Pull request closed without merging**